dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -96,7 +96,13 @@ DataInterfaceIIDM::build(std::string iidmFilePath) {
 
     std::string extensionsPaths = getMandatoryEnvVar("DYNAWO_LIBIIDM_EXTENSIONS");
     vector<string> paths;
-    boost::split(paths, extensionsPaths, boost::is_any_of(":"));
+    std::string splitCharacter;
+#ifdef _WIN32
+    splitCharacter = ";";
+#else
+    splitCharacter = ":";
```

Comment:
why not always using ;?

Reply:
to be consistent with what is done natively in each system, in windows we use `set PATH=path1;path2` and unix `export PATH=path1:path2`